### ISSUE TITLE
mban/multiban by regex

### ISSUE THREAD

**Issue by Pyxelx** [AEnterprise/GearBot#128]
mban by regex would be a nice feature, such as
`!mban regex SpamBot\d{4}` or something to ban any users that have the username like "SpamBot<4 numbers>"

shouldn't be too hard to implement, but idk much python.

**Comment by AEnterprise** [comment]
def something i can look into but it at the very least needs a massive warning of how many people it will ban (and probably a full list of them) to allow validating not too many people are locked in

**Comment by Rage0001** [comment]
Ooh, if the regex is faulty or something goes wrong with it, it could ban half of the server. People who don't know how regex works should be careful with this. Perhaps a warning as you said might be helpful.

**Comment by AEnterprise** [comment]
yeah this would have a warning of "you are about to ban 500 people, are you sure you really want to ban them all?" above a paged list of all users it would ban

**Comment by Pyxelx** [comment]
i think a warning in general would be a good idea, not just for faulty, but overall. "This regex is going to ban these members" or something like that.
